Arm/Insts/DPI/Add_sub_imm.lean

```diff
@@ -24,19 +24,21 @@ def exec_add_sub_imm (inst : Add_sub_imm_cls) (s : ArmState) : ArmState :=
                         else
                           imm <<< 12
     let operand1      := read_gpr datasize inst.Rn s
-    let (carry_in, operand2)
+    let carryInAndOperand2
```

Comment:
Could we leave a note (and an issue) to remind us to restore this definition to the original form after https://github.com/leanprover/lean4/issues/5388 is fixed?

Reply:
Done!